charts/vela-minimal/templates/defwithtemplate/bind-cloud-resource.yaml

```diff
@@ -0,0 +1,39 @@
+# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
+# Definition source cue file: vela-templates/definitions/internal/bind-cloud-resource.cue
+apiVersion: core.oam.dev/v1beta1
+kind: WorkflowStepDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: Sync secrets created by terraform component to runtime clusters
+  name: bind-cloud-resource
```

Comment:
In cloud resource `Provision and Binding`, `bind` means consuming/using the cloud resources by upper business application.  So the step name sounds a little bit confusing. The original `sync` is great, I think.

Reply:
I agree that `bind` is not a perfect name here but the `sync-cloud-resource` is also not very appropriate I think. How about `share-cloud-resource`?

Reply:
+1